adobecom/milo
# MWPW-143314: [Engineering][Universal Navigation][Milo] Add support for Notifications analytics

## Description
This PR adds analytics support for the universal navigation notifications

## Related Issue
Resolves: [MWPW-143314](https://jira.corp.adobe.com/browse/MWPW-143314)

## Testing instructions
1. go to the network tab in dev tools and search for 'collect';
2. click through the notifications part of the universal navigation;
3. you'll find the sent tag inside the payload, as you can see in the screenshot below;
<img width="1728" alt="Screenshot 2024-03-06 at 17 52 48" src="https://github.com/adobecom/milo/assets/146744221/369e1e04-37c7-4161-9c8b-b708fc336e26">

## Test URLs
**Acrobat:**
- Before: https://www.stage.adobe.com/acrobat/online/sign-pdf.html?martech=off
- After: https://www.stage.adobe.com/acrobat/online/sign-pdf.html?martech=off&milolibs=mwpw-143314-unav-notifications-analytics--milo--robert-bogos

**BACOM:**
- Before: https://main--bacom--adobecom.hlx.page/products/real-time-customer-data-platform/rtcdp?martech=off
- After: https://main--bacom--adobecom.hlx.page/products/real-time-customer-data-platform/rtcdp?martech=off&milolibs=mwpw-143314-unav-notifications-analytics--milo--robert-bogos

**CC:**
- Before: https://main--cc--adobecom.hlx.page/?martech=off
- After: https://main--cc--adobecom.hlx.page/?martech=off&milolibs=mwpw-143314-unav-notifications-analytics--milo--robert-bogos

**Homepage:**
- Before: https://main--homepage--adobecom.hlx.page/homepage/index-loggedout?martech=off
- After: https://main--homepage--adobecom.hlx.page/homepage/index-loggedout?martech=off&milolibs=mwpw-143314-unav-notifications-analytics--milo--robert-bogos

**Blog:**
- Before: https://main--blog--adobecom.hlx.page/
- After: https://main--blog--adobecom.hlx.page/?milolibs=mwpw-143314-unav-notifications-analytics--milo--robert-bogos

**Milo:**
- Before: https://main--milo--robert-bogos.hlx.page/drafts/rbogos/page-default?martech=off
- After: https://mwpw-143314-unav-notifications-analytics--milo--robert-bogos.hlx.page/drafts/rbogos/unav?martech=off
